models/evc.py

```diff
@@ -1154,35 +1154,36 @@ def check_trace(
                     circuits_checked
                 ):
         """Auxiliar function to check an individual trace"""
-        circuits_checked[circuit_id] = True
-        trace_a = circuit_by_traces[circuit_id]['trace_a']
-        trace_z = circuit_by_traces[circuit_id]['trace_z']
-        if len(trace_a) != len(circuit_current_path) + 1:
-            log.warning(f"Invalid trace from uni_a: {trace_a}")
-            circuits_checked[circuit_id] = False
-        if len(trace_z) != len(circuit_current_path) + 1:
-            log.warning(f"Invalid trace from uni_z: {trace_z}")
-            circuits_checked[circuit_id] = False
-        for link, trace1, trace2 in zip(circuit_current_path,
-                                        trace_a[1:],
-                                        trace_z[:0:-1]):
-            metadata_vlan = None
-            if link.metadata:
-                metadata_vlan = glom(link.metadata, 's_vlan.value')
-            if compare_endpoint_trace(
-                                        link.endpoint_a,
-                                        metadata_vlan,
-                                        trace2
-                                    ) is False:
+        if circuit_current_path:
```

Comment:
once the intra-switch EVCs will not be checked anymore, we should verify the list `circuits_checked` compared with `circuits_to_check` in `main.py`, to be redeployed again.

Reply:
@gretelliz I made a suggested change on commit 209d208 to address this point. Please let me know what you think.